tests/ibc-hooks/ibc_middleware_test.go

```diff
@@ -556,6 +566,12 @@ func (suite *HooksTestSuite) FullSend(msg sdk.Msg, direction Direction) (*sdk.Re
 		sender = suite.chainA
 	case CtoA:
 		sender = suite.chainC
+	case CW20toA:
+		sender = suite.chainB
+	case AtoCW20:
+		sender = suite.chainA
```

Comment:
nit: might be helpful for readability if we call this B-CW20 so the reviewer knows the CW chain is analogous to B

Reply:
I can rename it, but we can't use `-`. Any preference between `BCW20ToA`, `B_CW20ToA` or `Bcw20ToA`? They all look ugly to me